### PR TITLE
update spring #141

### PR DESCRIPTION
**Pull request by jobinhio**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/toolbox/pulls) for the same change?
- [x] Have you fixed a Bug ?
- [ ] Have you added the documentation of the example or benchmark ?
- [x] Have you successfully run the example or benchmark?
- [ ] Have you documented the results?
- [ ] Have you added the images to the documentation?
- [ ] Have you added the vtkjs files for 3D visualization in the documentation?
- [ ] Have you added the vtkjs files to a Data Management Plateform _e.g_ https://girder.math.unistra.fr

-----

### CODE REVIEW

File: examples/modules/csm/examples/spring/spring.json [hunk 1]
```diff
@@ -2,9 +2,29 @@
 {
   "Name": "Solid Mechanics ",
   "ShortName":"Solid",
+  "Metadata":
+  {
+      "version": "1",
+      "author": [""],
```
Comment (prudhomm) [inline examples/modules/csm/examples/spring/spring.json:8]: add authors see #145

**Review by prudhomm — CHANGES_REQUESTED** [review]
- the report of the simulation is missing  as well as the note regarding the report in the documentation see #136 
- add imagesprefix support as described in #144
- add authors to metadata #145

**Comment by jobinhio** [comment]
here are the results of displacement solid.
spring_displacement

here are the results of von mises criterion.
spring_von_mises_criterion

File: examples/modules/csm/pages/spring/README.adoc [hunk 2]
```diff
@@ -25,6 +27,7 @@ Copy case option to clipboard
 </button>
 ++++
 
+NOTE: The report of the execution of the command above is available xref:spring/solid-informations.adoc[here].
```
Comment (prudhomm) [inline examples/modules/csm/pages/spring/README.adoc:30]: informations -> information (rename the adoc file)

**Review by prudhomm — APPROVED** [review]

**Pull request merged**